File: Source/Fantasy.Jobs/TaskFailedException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Fantasy.Jobs
{
    [Serializable]
    public class TaskFailedException : Exception
    {
       public TaskFailedException(string message, Exception innerException)
            : base(message, innerException)
        {

        }

       public TaskFailedException(string message)
            : base(message)
        {

        }
    }
}

File: branches/Stheno/Fantasy.Jobs/src/fantasy/jobs/TaskFailedException.java
package fantasy.jobs;

import java.io.*;

public class TaskFailedException extends RuntimeException implements Serializable
{
   /**
	 * 
	 */
	private static final long serialVersionUID = 8741706106655930035L;

    public TaskFailedException(String message, RuntimeException innerException)
	{
	   super(message, innerException);

	}

   public TaskFailedException(String message)
	{
	   super(message);

	}
}